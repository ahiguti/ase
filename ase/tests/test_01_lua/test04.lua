
local Test = Native.Util.Test
local Test2 = Native.Util.Test
local UUID = Native.Util.UUID
local inst1 = Test();
local inst2 = Test();
local inst3 = UUID();

print("testclass:");
print(Test.IsUtilTestClass(Test));
print(Test.IsUtilTestClass(Test2));
print(Test.IsUtilTestClass(UUID));
print(Test.IsUtilTestClass(inst1));
print(Test.IsUtilTestClass(inst2));
print(Test.IsUtilTestClass(inst3));
print("testinstance:");
print(Test.IsUtilTestInstance(Test));
print(Test.IsUtilTestInstance(Test2));
print(Test.IsUtilTestInstance(UUID));
print(Test.IsUtilTestInstance(inst1));
print(Test.IsUtilTestInstance(inst2));
print(Test.IsUtilTestInstance(inst3));

