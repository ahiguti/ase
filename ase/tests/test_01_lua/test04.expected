testclass:
true
true
false
false
false
false
testinstance:
false
false
false
true
true
false
